fight/unit.js

```diff
@@ -0,0 +1,70 @@
+function Unit(name) {
+    this.MAXHEALTH = 100;// more fun with 1000
+    this._health = this.MAXHEALTH;
+    this._name =  name;
+    this._level = 1;
+    this._xp = 0;
+}
+Unit.prototype.getName = function(){
+    return this._name;
+}
+
+Unit.prototype.getLevel = function () {
+    return this._level;
+}
+Unit.prototype.isAlive = function () {
+    return !!this._health;

```

Comment:
I would use some more explicit way, just to be more you know, expressive

Reply:
it was a suggestion from webstorm

Reply:
`this._health > 0 ? true : false` , ternary is exceed, `<` return boolean already, `this._health > 0 ` is enough